src/main/java/edu/ucsb/cs56/ucsb_open_lab_scheduler/controllers/CourseOfferingController.java

```diff
@@ -46,18 +54,62 @@ public String dashboard(Model model, OAuth2AuthenticationToken token, RedirectAt
     }
 
     @PostMapping("/courseOffering/upload")
-    public String uploadCSV(@RequestParam("csv") MultipartFile csv, OAuth2AuthenticationToken token, RedirectAttributes redirAttrs) {
+    public String uploadCSV(@RequestParam("csv") MultipartFile csv, OAuth2AuthenticationToken token,
+            RedirectAttributes redirAttrs) {
         String role = authControllerAdvice.getRole(token);
         if (!(role.equals("Admin"))) {
             redirAttrs.addFlashAttribute("alertDanger", "You do not have permission to access that page");
             return "redirect:/";
         }
-        try(Reader reader = new InputStreamReader(csv.getInputStream())){
-            List<CourseOffering> courseOfferings =  csvToObjectService.parse(reader, CourseOffering.class);
+        try (Reader reader = new InputStreamReader(csv.getInputStream())) {
+            List<CourseOffering> courseOfferings = csvToObjectService.parse(reader, CourseOffering.class);
             courseOfferingRepository.saveAll(courseOfferings);
-        }catch(IOException e){
+        } catch (IOException e) {
             log.error(e.toString());
         }
-        return "redirect:/courseOffering";
+        return "redirect:/courseOffering/courseOffering";
```

Comment:
Looks like this redirect will cause an error, since this isn't a valid redirect path

Reply:
change made in branch